### PR TITLE
Contact Form

### PR DESCRIPTION
**Pull request by devalibello**
This pull request contains the first page of my portfolio with an input form for contact submission. It was done collaboratively with @fredo509 using peer programming

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has no descriptive PR title. Something like `Contact form` would have been the best PR title. Kindly change the PR title from `Pull Request of Feature collab2` to `Contact Form`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,142 +1,169 @@
...
+				<section class="contact">
+					<h2 class="contact-me">Contact me</h2>
+					<p class="info-contact">If you have an application you are interested in developing, a feature that
+						you need built or a project that needs coding. I’d love to help with it </p>
+					<form action="https://formspree.io/f/mvonabpr" method="post" class="form-input">
+						<input class="name" type="text" id="" maxlength="30" placeholder="Enter your name" required>
+						<input class="name" type="email" id="" placeholder="Enter your email" required>
+						<textarea class="name" name="" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+						<input type="submit" value="Get in touch" name="user_name" class="get_in_touch">
+					</form>
```
Comment (wayungi) [inline index.html]: - [ ] At 375px with, your form touches both ends of the screen.
- [ ] Your form overflows the width of the device. These issues can be resolved by assigning relative width in percentages. 
**figma**
width1

**Your design**
width2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @devalibello , 
image

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (devalibello) [reply index.html]: @wayungi Hello, I will like more guidance as to how to solve this issue. As you mentioned, I am already using a width in percentage `width: 90%;` from the css styling `.name {...}`

Many thanks

**Review by binhussen — APPROVED** [review]
Hi @devalibello , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**